src/runtime/FindFilesystem.cmake

```diff
@@ -0,0 +1,287 @@
+# Distributed under the OSI-approved BSD 3-Clause License.
+# See https://cmake.org/licensing for details.
+# This file is part of CMake Community Modules:
+# https://github.com/vector-of-bool/CMakeCM/blob/master/modules/FindFilesystem.cmake
+#[=======================================================================[.rst:
+
+FindFilesystem
```

Comment:
I wonder if this file might better be placed in our `cmake` directory?

Reply:
Done :ok_hand: